README.md

```diff
@@ -83,7 +83,7 @@ Use [ROS](http://wiki.ros.org/ROS/Installation) or [ROS 2](https://index.ros.org
 Once you have the desired sources, copy them in the `sysroot` to use with the tool.
 ```bash
 # Copy ros sources into the sysroot directory
-cp -r <full_path_to_your_ros_ws>/src sysroot/ros_ws/src
+cp -r <full_path_to_your_ros_ws>/src sysroot/ros_ws
```

Comment:
Is this really correct? if `ros_ws` already exists, then yes it will create `sysroot/ros_ws/src`, but if it doesn't then `sysroot/ros_ws` will contain the contents of `src`, yeah?

Reply:
There is a step to create `sysroot/ros_ws/src` [before](https://github.com/ros-tooling/cross_compile/pull/146/files/f924b3b0e9f1e83ee6e55e397ec08957c82efda5#diff-04c6e90faac2675aa89e2176d2eec7d8R70) this one.

Reply:
Ok go ahead then. Looks like you need to update your branch